config/locales/en.yml

```diff
@@ -603,6 +603,7 @@ en:
         description: 'Supports HTML'
         coach_spaces: Available coach spaces
         student_spaces: Available student spaces
+        invitable: When checked, it overrides the RSVP open local date and time.
```

Comment:
@despo I think we need to be more explicit here and say that this checkbox overrides the RSVP open local date and time and opens RSVP's. I think we need to mention that it actually opens RSVP's right away.

Reply:
updated text